packages/angular_devkit/build_angular/package.json

```diff
@@ -68,6 +69,7 @@
     "terser-webpack-plugin": "5.1.2",
     "text-table": "0.2.0",
     "tree-kill": "1.2.2",
+    "tslib": "2.2.0",
```

Comment:
Let's pin the one in the root `package.json` since Bazel doesn't support workspaces.